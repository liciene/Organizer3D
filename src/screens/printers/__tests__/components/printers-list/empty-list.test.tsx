import { shallow } from 'enzyme';
import * as React from 'react';

import EmptyList from '../../../components/printers-list/empty-list';

describe('=== PRINTERS === COMPONENTS === PrinterList --- EmptyList', () => {
    // #region Configurations

    const wrapper = shallow(<EmptyList />);

    // #endregion

    // #region should render and is equal

    it('should render and is equal', () => {
        expect(wrapper).toMatchSnapshot();
    });

    // #endregion
});
