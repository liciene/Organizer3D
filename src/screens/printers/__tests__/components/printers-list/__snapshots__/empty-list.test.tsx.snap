// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`=== PRINTERS === COMPONENTS === PrinterList --- EmptyList should render and is equal 1`] = `
<View
  style={
    Object {
      "backgroundColor": "rgba(0,0,0,0.5)",
      "height": 1334,
      "padding": 12,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
      }
    }
  >
    <Icon
      allowFontScaling={false}
      color="#fff"
      name="emoticon-sad-outline"
      size={64}
    />
    <Text
      style={
        Object {
          "color": "#fff",
          "fontSize": 32,
          "textAlign": "center",
        }
      }
    >
      The list is empty, please add a printer
    </Text>
  </View>
  <View
    style={
      Object {
        "backgroundColor": "#fff",
        "borderRadius": 200,
        "bottom": 10,
        "height": 200,
        "position": "absolute",
        "right": -50,
        "width": 200,
      }
    }
  />
</View>
`;
